bigtable/google/cloud/bigtable/table.py

```diff
@@ -386,6 +386,47 @@ def sample_row_keys(self):
             self.name)
         return response_iterator
 
+    def truncate(self, timeout=60):
```

Comment:
The timeout is too low.  Isn't there GAPIC configuration for the default value?  If so, please use that.  The minimum timeout for this method is 2 minutes.  I'm not sure how 60 seconds works here.

Have you run a live test with this method?  If you haven't please do so to see the results. Please add a system test, either way.